index.html

```diff
@@ -266,8 +281,10 @@ <h3 class="heading5">Professional Art Printing Data</h3>
           <a class="icon2" href="#"><img src="images/ruby1.png" alt="ruby" /></a>
         </li>
       </ul>
-
-      <button class="button2">See Project</button>
+      <div class=buttonlink onclick="modal(3)">
+        <button class="button2">See Project</button>
+      </div>
+      
```

Comment:
- [ ] Great effort working on the changes requested by the previous reviewer, According to the previous reviewer, he suggested you remove all the card sections you have in your `HTML` file `(from line 144 to 288)` as this has been created dynamically. removing it will make you be sure that your card section is created properly dynamically. Please kindly remove them and with this, you stick to the project requirement.

   - Project requirement
   <img src="https://user-images.githubusercontent.com/77843050/210660442-a080dd88-e79d-48ac-9fcc-225d13d9231f.png">